sphinx/ext/todo.py

```diff
@@ -212,8 +218,14 @@ def depart_todo_node(self, node):
 
 def latex_visit_todo_node(self, node):
     # type: (nodes.NodeVisitor, todo_node) -> None
-    title = node.pop(0).astext().translate(tex_escape_map)
-    self.body.append(u'\n\\begin{sphinxadmonition}{note}{%s:}' % title)
+    target = node.get('targetref')
```

Comment:
Could you move this before if-block?

Reply:
Sorry, I don't understand this comment. As far as I can see, it is before the if-block. Maybe I'm misunderstanding which if-block you're referring to?

Reply:
Sorry, I think `target` is only related with outputting a label. So I prefer to move this line to just above of if-block. In other word, four lines bottom from here (after comment).